test/features/pagecontent/save_api.js

```diff
@@ -418,6 +418,7 @@ describe('page save api', function() {
                     uri: htmlUri,
                     headers: {
                         'x-forwarded-for': '123.123.123.123',
+                        'x-client-ip': '123.123.123.123',
```

Comment:
@d00rman This is not what we set, but what we expect, it's a `nock` configuration.